images/twitter.css

```diff
@@ -0,0 +1,314 @@
+*{
+    font-family: Arial, Helvetica, sans-serif;
+}
+.bigcontainer{
+    display: grid;
+    grid-template-columns: 5fr 10fr 5fr;
+    grid-template-areas: "left middle right";
+    background-color: whitesmoke;
+    gap: 5px;
+}
+.leftcontainer{
+    grid-area: left;
+    border: 1px solid grey;
+    height: 750px;
+    width: 
+    gap: 5px;
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-areas:
+    "leftlogo"
+    "home"
+    "explore"
+    "notifications"
+    "messages"
+    "bookmarks"
+    "lists"
+    "profiles"
+    "more"
+    "bottomprofile"
+    ;
+}
+.leftcontainer p{
+   margin: 10px;
+}
+    
+.middlecontainer{
+    border: 1px solid grey;
+    grid-area: middle;
+    height: 750px;
+    gap: 5px;
+    display: grid;
+    grid-template-rows: 0.5fr 1fr 1fr 2fr;
+    grid-template-columns: 1fr;
+    grid-template-areas:
+    "header"
+    "topmid"
+    "midmid"
+    "lowmid";
+}
+.header{
+    grid-area: header;
+    padding: 10px;
+    margin-top: 10px;
+}
+.topmidprofile{
+    display: grid;
+    border: 1px solid grey;
+    grid-area: topmid;
+    grid-template-rows: 1fr 1fr;
+    grid-template-areas: 
+    "topmid1"
+    "topmid2";
+}
+.topmid1{
+    grid-area: topmid1;
+    display: grid;
+    grid-template-columns: 1fr 20fr;
+}
+.topmid1 p{
+   color: grey; 
+   font-size: small;
+}
+.topmid2{
+    grid-area: topmid2;
+    display: grid;
+    grid-template-columns: 0.1fr 0.1fr 0.1fr 0.1fr 0.1fr 3fr;
+    margin-left: 50px;
+    grid-template-rows: 1fr;
+    grid-template-areas:
+    "topmessageoption1 topmessageoption2 topmessageoption3 topmessageoption4 topmessageoption5 ."
+}
+.topmid2 img{
+    width: 10px;
+}
+.topmessageoption1{
+    grid-area: topmessageoption1;
+}
+.topmessageoption2{
+    grid-area: topmessageoption2;
+}
+.topmessageoption3{
+    grid-area: topmessageoption3;
+}
+.topmessageoption4{
+    grid-area: topmessageoption4;
+}
+.topmessageoption5{
+    grid-area: topmessageoption5;
+}
+.topmid2 .topoptiontweet p{
+    background-color: lightblue;
+    border-radius: 20px;
+    color: white;
+    text-align: center;
+    padding: 10px;
+    font-size: small;
+    display: inline;
+    justify-content: right;
+    margin: 20px;
+    font-weight: bold;
+}
+.midmidprofile{
+    border: 1px solid grey;
+    grid-area: midmid;
+    display: grid;
+    grid-template-columns: 1fr ;
+    grid-template-rows: 2fr 1fr;
+    grid-template-areas: 
+    "midmid1"
+    "midmid2";
+}
+.small.para{
+    font-weight: lighter;
+}
+.midmidprofile .midmid1 p{
+    font-size: x-small;
+    margin: 10px;
+    padding: ;
+    text-align: left;
+}
+.midmid1 .someoneretweeted{
+    color: grey;
+}
+.midmid1{
```

Comment:
I think this is the part I was looking for. for your post you could create an empty column and then put the para or you could make the para take up both columns in the post. 

kinda like " .  para" if you want the space under the profile pic to be blank, or "para para"  so that it goes across

![for hamza](https://user-images.githubusercontent.com/112327857/206681197-a8362207-3505-457b-8e08-6cdebb21a6ae.jpg)